project_euler/problem_56/sol1.py

```diff
@@ -0,0 +1,15 @@
+def maximum_digital_sum(a,b):
```

Comment:
Please consider adding Python [__type hints__](https://docs.python.org/3/library/typing.html) like: __def maximum_digital_sum(a: int, b: int) -> int:__ so that __mypy__  can do type checking in our automated testing.

Also, please consider adding Python [__doctests__](https://docs.python.org/3/library/doctest.html) so that __pytest__ can run those tests in our automated testing.

https://github.com/TheAlgorithms/Python/blob/master/.travis.yml